pennylane/_qubit_device.py

```diff
@@ -564,8 +564,8 @@ def estimate_probability(self, wires=None):
         samples = self._samples[:, device_wires]
 
         # convert samples from a list of 0, 1 integers, to base 10 representation
-        unraveled_indices = [2] * len(device_wires)
-        indices = np.ravel_multi_index(samples.T, unraveled_indices)
+        powers_of_two = 2 ** np.arange(len(device_wires))[::-1]
```

Comment:
Would `1 << np.arange(...)` be ever so slightly faster too?

Reply:
oooh, maybe! I'm not sure

Reply:
The answer seems to be "yes but barely"
![image](https://user-images.githubusercontent.com/8702042/108231697-ddc6ef80-70f6-11eb-9622-2904abf0e64f.png)

Reply:
Worth it? comes down to barely speed increase vs code legibility 🤔

Reply:
yeah not worth it.